### PR TITLE
Remove ideaSpaceToSchoolClass and use lenses instead

### PR DESCRIPTION
**Pull request by np**

### CODE REVIEW

File: src/DemoData.hs [hunk 1]
```diff
@@ -202,7 +202,7 @@ universe rnd size = do
 
     ideaSpaces <- nub <$> generate (numberOfIdeaSpaces size) rnd arbitrary
     mapM_ (update . AddIdeaSpaceIfNotExists) ideaSpaces
-    let classes = mapMaybe ideaSpaceToSchoolClass ideaSpaces
+    let classes = ideaSpaces ^.. each . _ClassSpace
```
Comment (andorp) [inline src/DemoData.hs]: https://hackage.haskell.org/package/lens-4.14/docs/src/Control.Lens.Fold.html#%5E..

**Comment by andorp** [comment]
LGTM

**Comment by fisx** [comment]
LGTM, too.  @np, could you make travis happy and merge?

I would also like to have the link into the lens docs in the code, but I suspect you disagree there.  (-:

**Comment by np** [comment]
Sorry for wishful coding... @andorp the naming convention was to call it Erratatata though :)

**Pull request merged**